dockerfiles/gnupg/Dockerfile

```diff
@@ -0,0 +1,34 @@
+ARG VCS_REF=master
+ARG BUILD_DATE=""
+ARG REGISTRY_PATH=docker.io/paritytech
+
+# 'Parity Security Team <security@parity.io>'
+ARG GPG_KEYID=9D4B2B6EB8F97156D19669A9FF0812D491B96798
+
+FROM docker.io/library/ubuntu:latest
+
+# metadata
+LABEL summary="Base image for GnuPG operations" \
+	name="${REGISTRY_PATH}/gnupg" \
+	maintainer="devops-team@parity.io" \
+	version="1.0" \
+	description="GnuPG base container" \
+	io.parity.image.vendor="Parity Technologies" \
+	io.parity.image.source="https://github.com/paritytech/scripts/blob/${VCS_REF}/dockerfiles/gnupg/Dockerfile" \
+	io.parity.image.documentation="https://github.com/paritytech/scripts/blob/${VCS_REF}/dockerfiles/gnupg/README.md" \
```

Comment:
please consider adding small READMEs, with general info what's there and why it's needed. We will be upstreaming them to dockerhub image descriptions.